sf-docker/jetson-docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       - AppSettings__USE_JAEGER_APP_SETTINGS
       - JAEGER_AGENT_HOST
       - S3Bucket__Endpoint
-      - GstPipelineTemplate
+      - GstPipelineTemplate_Jetson
```

Comment:
`- GstPipelineTemplate=${GstPipelineTemplate_Jetson}` needs to be here for it to work